wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/StructTest.kt

```diff
@@ -349,53 +317,49 @@ class StructTest {
   }
 
   @Test fun structJsonRoundTripWithEmptyOrNestedMapAndList() {
-    val json = """{
-         |  "struct": {"a": null},
-         |  "list": [],
-         |  "valueA": {
-         |    "a": [
-         |      "b",
-         |      2.0,
-         |      {"c": false}
-         |    ]
-         |  },
-         |  "valueB": [{"d": null, "e": "trois"}],
-         |  "valueC": [],
-         |  "valueD": {},
-         |  "valueE": null,
-         |  "valueF": null
-         |}""".trimMargin()
-    // Wire prints null value members while protoc doesn't.
-    val jsonWithNullValue = """{"nullValue": null, ${json.substring(1)}"""
+    val protocJson = """{
+      |  "struct": {"a": null},
+      |  "list": [],
+      |  "valueA": {"a": ["b", 2.0, {"c": false}]},
+      |  "valueB": [{"d": null, "e": "trois"}],
+      |  "valueC": [],
+      |  "valueD": {},
+      |  "valueE": null,
+      |  "valueF": null
+      |}""".trimMargin()
+    // Protoc doesn't print those unless explicitly set.
```

Comment:
Hmmmm .. . . is there a semantic difference here? I wonder if these two are equals()